src/OverviewPage/OverviewPage.tsx

```diff
@@ -0,0 +1,312 @@
+import React from 'react';
+import {
+    Bullseye,
+    Button,
+    ButtonVariant,
+    Card,
+    CardBody,
+    CardFooter,
+    CardHeader,
+    CardHeaderMain,
+    CardTitle,
+    Grid,
+    Label,
+    PageSection,
+    PageSectionVariants,
+    Stack,
+    StackItem,
+    Text,
+    TextVariants,
+    Title,
+} from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+import ApacheKafkaLogo from './static/images/Logo-Red_Hat-OpenShift_Streams_for_Apache_Kafka-A-Standard-RGB.svg';
+import APIMgmtLogo from './static/images/Logo-Red_Hat-OpenShift-API_Management-A-Standard-RGB.svg';
+import DataScienceLogo from './static/images/Logo-Red_Hat-OpenShift-Data_Science-A-Standard-RGB.svg';
+import ServiceRegistryLogo from './static/images/Logo-Red_Hat-OpenShift_Service_Registry-A-Standard-RGB.png';
+import DataServicesLogo from './static/images/Logo-Red_Hat-Data_Services-A-Standard-RGB.png';
+import './OverviewPage.css'
+
+import { useTranslation } from 'react-i18next';
+
+export type OverviewPageProps = {
+    onClickKafkainstance: (event: any) => void;
+    getKafkaHref: string;
```

Comment:
```suggestion
    toKafkaHref: string;
```

`get` implies an action, I'd expect a function to call, not a primitive type